instructions/Lab1.md

```diff
@@ -89,7 +89,7 @@ kubectl run nginx --image=nginx --port=80 --dry-run=client -o yaml > nginx.yaml
 ```
 </p></details>
 
-### 7. Rewrite the yaml file from a pod configuration to a deployment named "nginx-deployment" with 2 replicas
+### 7. Rewrite the yaml file from a pod configuration to a deployment named "nginx-deployment" with 2 replicas <a id="rewrite-deployment"></a>
```

Comment:
for what is this added link? this is not displayed in the rendered md file on github

Reply:
Ah I see, this is because you want to reference this from somewhere else